notebooks/book2/03/hierarchical_binom_rats.ipynb

```diff
@@ -0,0 +1,1831 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        [
</pre><p>Can you reformat this data vector so it takes less space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>